From ed2cac0f1e346cfb01e353071c91331485af5538 Mon Sep 17 00:00:00 2001
From: Yiwei Zhang <zzyiwei@chromium.org>
Date: Wed, 11 Oct 2023 20:07:03 +0000
Subject: [PATCH 192/834] venus: enable Vulkan 1.3 for Android 13 and above

Now that ndk translation support is ready for both Intel and AMD, we can
advertise Vulkan 1.3 on Android 13.

Signed-off-by: Yiwei Zhang <zzyiwei@chromium.org>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/25666>
---
 src/virtio/vulkan/vn_instance.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/virtio/vulkan/vn_instance.h b/src/virtio/vulkan/vn_instance.h
index bd64f28af89..e2784579bec 100644
--- a/src/virtio/vulkan/vn_instance.h
+++ b/src/virtio/vulkan/vn_instance.h
@@ -25,7 +25,7 @@
 #define VN_MIN_RENDERER_VERSION VK_API_VERSION_1_1
 
 /* max advertised version at both instance and device levels */
-#ifdef ANDROID
+#if defined(ANDROID) && ANDROID_API_LEVEL < 33
 #define VN_MAX_API_VERSION VK_MAKE_VERSION(1, 1, VK_HEADER_VERSION)
 #else
 #define VN_MAX_API_VERSION VK_MAKE_VERSION(1, 3, VK_HEADER_VERSION)
-- 
2.42.0

